packages/s/sfml/xmake.lua

```diff
@@ -0,0 +1,116 @@
+package("sfml")
+
+    set_homepage("https://www.sfml-dev.org")
+    set_description("Simple and Fast Multimedia Library")
+
+    if is_plat("windows") then
+        if is_arch("x64") then
+            set_urls("https://www.sfml-dev.org/files/SFML-$(version)-windows-vc15-64-bit.zip")
+            add_versions("2.5.1", "3e807f7e810d6357ede35acd97615f1fe67b17028ff3d3d946328afb6104ab86")
+        elseif is_arch("x86") then
+            set_urls("https://www.sfml-dev.org/files/SFML-$(version)-windows-vc15-32-bit.zip")
+            add_versions("2.5.1", "9c7bef70ef481884756c9b52851c73caea11eeacb5cc83d03a3c157aee9e395f")
+        end
+    elseif is_plat("linux") then
+        if is_arch("x64", "x86_64") then
+            set_urls("https://www.sfml-dev.org/files/SFML-$(version)-linux-gcc-64-bit.tar.gz")
+            add_versions("2.5.1", "34ad106e4592d2ec03245db5e8ad8fbf85c256d6ef9e337e8cf5c4345dc583dd")
+        end
+    elseif is_plat("macosx") then
+        if is_arch("x64", "x86_64") then
+            set_urls("https://www.sfml-dev.org/files/SFML-$(version)-macOS-clang.tar.gz")
+            add_versions("2.5.1", "6af0f14fbd41dc038a00d7709f26fb66bb7ccdfe6187657ef0ef8cba578dcf14")
+        end
+    elseif is_plat("mingw") then
+        if is_arch("x64", "x86_64") then
+            set_urls("https://www.sfml-dev.org/files/SFML-$(version)-windows-gcc-7.3.0-mingw-64-bit.zip")
+            add_versions("2.5.1", "671e786f1af934c488cb22c634251c8c8bd441c709b4ef7bc6bbe227b2a28560")
+        elseif is_arch("x86") then
+            set_urls("https://www.sfml-dev.org/files/SFML-$(version)-windows-gcc-7.3.0-mingw-32-bit.zip")
+            add_versions("2.5.1", "92d864c9c9094dc9d91e0006d66784f25ac900a8ee23c3f79db626de46a1d9d8")
+        end
+    end
+
+    add_configs("shared",     {description = "Uses the shared libraries (requires .dll or .so)", default = true, type = "boolean"})
```

Comment:
This is not needed, shared is a built-in option

Reply:
I wanted to set its default value to ```true```, but it wasn't affected.

Reply:
This will break the default convention rules of add_requires.

```lua
add_requires("xxx") -- always use static libs by default
add_requires("xxx", {configs = {shared = true}}) 
```